chroma-manager/tests/framework/utils/install_deps.sh

```diff
@@ -4,4 +4,5 @@ mkdir -p chroma_test_env
 tar -C chroma_test_env -xizf -"
 
 ssh root@$CHROMA_MANAGER "set -ex
-yum -y install nodejs npm nginx libuv"
+yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/repo/epel-7/managerforlustre-manager-for-lustre-epel-7.repo
+yum -y install nodejs npm nginx libuv iml-gui iml-srcmap-reverse"
```

Comment:
This is actually because the base was set to master instead of the extract-gui patch (which is still waiting on your review).

Let's continue discussion on this over there.